fix: quickstart tweaks (#38)
web3-storage/www: src/pages/docs/quickstart.md

You need to create a web3.storage account associated with an email address and set it up so you can start uploading to a Space. The Space is created locally and associated with a private key, and is then registered with web3.storage and associated with your email address. But don't worry about keeping track of the Space's private key! web3.storage's email authorization system allows this private key to be treated as a throwaway key.

> 1. Run `w3 login [alice@example.com](mailto:alice@example.com)` in the command line using your email address. This will sent an email to your inbox with a link for validation.

Author: I found this confusing and unecessary

Reviewer: Do we want a simple code example like a few lines below? So people know the syntax?
something like: w3 login [your email]

Author: I thought so first, but did not want to break the listing order thing

> 2. Once you click on the validation link, you'll be taken to a webpage where you can enter your payment information and select a plan (like our Free tier).

Author: We dont need to be this explicit in the docs I believe. Thoughts?

Reviewer: agreed. we should also change "free" to "starter"

https://w3s.link/ipfs/bafybeib4ht2a53pttgipw6mgckqqhmgkifnmh2glzsju2c6ait5ibnkow4

> By default, `w3` will wrap uploaded files into a folder, so that filename is preserved. It can then be accessed directly by adding the name in the URL path:

Reviewer 2: ```suggestion
By default, `w3` will wrap files in a folder, so that their filename is preserved. They can then be accessed directly by adding their name in the URL path:
```
